### ISSUE TITLE
Poor training performance of delg

### ISSUE THREAD

**Issue by himsR** [tensorflow/models#9176]
Hello,
I cloned the delg repo and tried to train delg on landmark clean -v2  with the default resnet-50 backbone for a while on a 2- gpu machine (TITAN-V+1080 ti)
for several iterations(batch size - 26, lr-.001). It seems that the model is not learning anything at all. My output is somthing like:

Validation(3000.000000)

I0831 23:19:53.560219 139707578869504 train.py:519] : desc: 0.000020

I0831 23:19:53.560388 139707578869504 train.py:520] : attn: 0.000020

Eventually these become zero too.
I have also attached my losses from tensorboard. The command line argument I use torun delg is :

--train_file_pattern=/media/hims/69c57ccf-aa7f-4a0a-bf49-539a27acad35//tfrecord/train*
--validation_file_pattern=/media/hims/69c57ccf-aa7f-4a0a-bf49-539a27acad35/tfrecord/validation*
--imagenet_checkpoint=resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5
--dataset_version=gld_v2_clean
--logdir=gldv2_training/
--delg_global_features
Am I missing something ? 
![delg](https://user-images.githubusercontent.com/6899625/91795276-3955fd00-ebeb-11ea-987c-df56e96f335c.png)

**Comment by andrefaraujo** [comment]
Hi @himsR , thanks for trying out the DELG training code. This is still work-in-progress to a good extent, stay tuned as more updates will come in the following weeks.

Regarding what you observed: @dan-anghel has also noticed some issues when training, depending on the learning rate, batch size, number of GPUs, etc. In particular, recently when training with 4 GPUs, batch size of 128, decreasing the initial learning to 0.005 was necessary.

Dan, what do you think of providing some guidelines in the README about some default training parameters that should work on GLDv2-clean with the default instructions?

**Comment by dan-anghel** [comment]
Hi @himsR , @andrefaraujo. I created the following PR which updates the documentation with hyperparameter values we tested on 8xP100 and 4xP100 machines: https://github.com/tensorflow/models/pull/9219.
